src/components/inputs/reactHookForm/utils/HelperPreviousValue.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { useIntl } from 'react-intl';
+import { FormHelperText, Stack, Tooltip, Typography } from '@mui/material';
+import { InfoOutlined, WarningAmberRounded } from '@mui/icons-material';
+
+export interface HelperPreviousValueProps {
+    previousValue?: number | string;
+    isNodeBuilt?: boolean;
+    disabledTooltip?: boolean;
+    adornmentText?: string;
+}
+
+export function HelperPreviousValue({
+    previousValue,
+    isNodeBuilt,
+    disabledTooltip,
+    adornmentText,
+}: Readonly<HelperPreviousValueProps>) {
+    const intl = useIntl();
+    if (previousValue || previousValue === 0) {
```

Comment:
I think this is clearer : 
```suggestion
  if (previousValue !== undefined) {
```
But I think starting like this would be even clearer : 
```
if (previousValue === undefined) {
    return undefined;
}
```
And then the regular return.

In fact given that `previousValue` is absolutely needed you could make it mandatory (without `?`) and do the check in the components using `HelperPreviousValue`.